addons/web/views/webclient_templates.xml

```diff
@@ -262,6 +264,7 @@
         <script type="text/javascript" src="/web/static/src/js/tools/test_menus_loader.js"/>
         <script type="text/javascript" src="/web/static/src/js/tools/debug_manager_backend.js"></script>
         <script type="text/javascript" src="/web/static/src/js/tools/tools.js"></script>
+        <script type="text/javascript" src="/web/static/src/js/env.js"></script>
```

Comment:
put an env.js file in assets frontend with services.rpc, qweb... (mandatory stuff)